docs/sdk/web/adapters/torus-solana.mdx

```diff
@@ -108,6 +108,7 @@ If you do pass `chainConfig` in an Adapter, it overwrites the `chainConfig` pass
 import { SolanaWalletAdapter } from "@web3auth/torus-solana-adapter";
 const solanaWalletAdapter = new SolanaWalletAdapter({
   adapterSettings: {
+    clientId: "YOUR_CLIENT_ID",
```

Comment:
Let's change these with `YOUR_WEB3AUTH_CLIENT_ID` to be consistent in docs. There are other client id's as well such as from different providers.